docs/API.md

```diff
@@ -1385,6 +1385,7 @@ __Parameters__
 | `object_name`        | _str_            | Object name in the bucket.                 |
 | `ssec`               | _SseCustomerKey_ | Server-side encryption customer key.       |
 | `version_id`         | _str_            | Version ID of the object.                  |
+| `extra_headers`    | _dict_           | Any additional headers to be added with GET request. |
```

Comment:
Please fix the doc
1. Align the table
2. Have description as `Extra headers for advanced usage.`